pkg/apis/acme/v1/types_issuer.go

```diff
@@ -114,6 +114,11 @@ type ACMEIssuer struct {
 	// Defaults to false.
 	// +optional
 	EnableDurationFeature bool `json:"enableDurationFeature,omitempty"`
+
+	// Profile allows requesting a certificate profile from the ACME server.
+	// Supported profiles are listed by the server's ACME directory URL.
+	// +optional
```

Comment:
Perhaps I should expand this documentation to explain how cert-manager will behave when the profile is empty and how it will behave if the profile is not supported by the ACME server.

Reply:
Yes, the random profile selection might be surprising, would be good to add docs about this.